/**:
  ros__parameters:
    # フィルタの種類: "average", "median", "none" から選択
    filter_type: "average"
    # 移動平均/中央値フィルタのウィンドウサイズ
    window_size: 5
    # スケール/クリップフィルタを有効にするか
    use_scale_filter: true
    # スケールフィルタの種類: "normal" or "advance"
    scale_filter_type: "normal"

    # ---------------------------------------------------
    # "normal" モード用のパラメータグループ
    # ---------------------------------------------------
    normal:
      speed_scale_ratio: 1.0
      steer_scale_ratio: 1.0

    # ---------------------------------------------------
    # "advance" モード用のパラメータグループ
    # ---------------------------------------------------
    advance:
      # 直進状態と判断する操舵角の閾値 (単位: rad)
      straight_steer_threshold: 0.075
      # ★★★ "直進時"の速度スケール比
      straight_speed_scale_ratio: 1.0
      # カーブ時の速度スケール比
      cornering_speed_scale_ratio: 1.0
      # 操舵角のスケール比
      steer_scale_ratio: 1.0